tests/cmdline/commands/test_storage.py

```diff
@@ -176,3 +179,13 @@ def mock_maintain(*args, **kwargs):
     assert ' > full: True' in message_list
     assert ' > do_repack: False' in message_list
     assert ' > dry_run: False' in message_list
+
+
+def tests_storage_backup(run_cli_command):
+    """Test the ``verdi storage backup`` command."""
+    with tempfile.TemporaryDirectory() as tmpdir:
+        result = run_cli_command(cmd_storage.storage_backup, parameters=[tmpdir])
+        assert ' backed up to ' in result.output
+        assert result.exit_code == 0
+        last_backup = Path(tmpdir) / 'last-backup'
+        assert last_backup.is_symlink()
```

Comment:
for temporary paths in unit tests, we should just use `pytest`'s `tmp_path` fixture:
```suggestion
def tests_storage_backup(run_cli_command, tmp_path):
    """Test the ``verdi storage backup`` command."""
    result = run_cli_command(cmd_storage.storage_backup, parameters=[str(tmp_path)])
    assert ' backed up to ' in result.output
    assert result.exit_code == 0
    last_backup = tmp_path / 'last-backup'
    assert last_backup.is_symlink()
```